process/block/metrics.go

```diff
@@ -126,6 +126,14 @@ func incrementCountAcceptedBlocks(
 	}
 
 	bitMap := header.GetPubKeysBitmap()
+	indexOutOfBounds := myIndex/8 >= len(bitMap)
+	if indexOutOfBounds {
+		log.Debug("process blocks metrics: index out of bounds",
```

Comment:
Trace ?

Reply:
This shouldn't happen too often, but ok, I'll move it to trace